Mathlib/Order/KrullDimension.lean

```diff
@@ -135,18 +181,34 @@ lemma length_le_height {p : LTSeries α} {x : α} (hlast : p.last ≤ x) :
     simp [p']
   · simp_all
 
+lemma length_le_coheight {x : α} {p : LTSeries α} (hhead : x ≤ p.head) :
+    p.length ≤ coheight x :=
+  length_le_height (α := αᵒᵈ) (p := p.reverse) (by simpa)
+
 /--
 The height of the last element in a series is larger or equal to the length of the series.
 -/
 lemma length_le_height_last {p : LTSeries α} : p.length ≤ height p.last :=
   length_le_height le_rfl
 
+/--
+The coheight of the first element in a series is larger or equal to the length of the series.
+-/
+lemma length_le_coheight_last {p : LTSeries α} : p.length ≤ coheight p.head :=
```

Comment:
```suggestion
lemma length_le_coheight_head {p : LTSeries α} : p.length ≤ coheight p.head :=
```
... unless I've lost my last, hmm head

Reply:
No, certainly my mind that got lost at some point here :-)